engine/src/main/java/org/terasology/math/AABB.java

```diff
@@ -88,6 +107,12 @@ public static AABB createEncompassing(Iterable<AABB> others) {
         return new AABB(min, max);
     }
 
+    /**
+     * Creates a new AABB that contains the vertices as represented by a {@link TFloatList}.
+     *
```

Comment:
Important information for a caller would be the format for vertices and what would be the result. An example would be nice. E.g. this method spans a bounding box which contains all vertices.